project/Http4sPlugin.scala

```diff
@@ -26,20 +26,9 @@ object Http4sPlugin extends AutoPlugin {
     },
     scalaVersion := (sys.env.get("TRAVIS_SCALA_VERSION") orElse sys.env.get("SCALA_VERSION") getOrElse "2.12.3"),
 
-    scalacOptions in Compile ++= Seq(
-      "-Yno-adapted-args", // Curiously missing from RigPlugin
-      "-Ypartial-unification", // Needed on 2.11 for Either, good idea in general
-      "-Ywarn-unused-import"
-    ) ++ {
-      // https://issues.scala-lang.org/browse/SI-8340
-      CrossVersion.partialVersion(scalaVersion.value) match {
-        case Some((2, n)) if n >= 11 => Seq("-Ywarn-numeric-widen")
-        case _ => Seq.empty
-      }
-    },
-    scalacOptions in (Compile, console) ~= { xs: Seq[String] =>
-      xs.filterNot(Set("-Xfatal-warnings", "-Ywarn-unused-import"))
-    },
+    scalacOptions --= Seq(
+      "-Ywarn-unused:patvars"
+    ),
```

Comment:
I'd like to get rid of this one too and do the full tpolecat.  I was just overwhelmed at first.